boards/samr34-xpro/board.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_samr34-xpro
+ * @{
+ *
+ * @file        board.c
+ * @brief       Board specific implementations for the Microchip
+ *              SAM R34 Xplained Pro board
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "cpu.h"
+#include "periph/gpio.h"
+
+#ifdef MODULE_SX127X
+#include "sx127x_params.h"
+#endif
+
+void led_init(void);
+
+void board_init(void)
+{
+    /* initialize the CPU */
+    cpu_init();
+
+    /* initialize the boards LEDs */
+    led_init();
+
+    /* initialize board specific pins for LoRa */
+#ifdef MODULE_SX127X
+    gpio_init(TXCO_PWR_PIN, GPIO_OUT);
+    gpio_set(TXCO_PWR_PIN);
+    gpio_init(TX_OUTPUT_SEL_PIN, GPIO_OUT);
+    gpio_write(TX_OUTPUT_SEL_PIN, !sx127x_params[0].paselect);
```

Comment:
Why not directly use `SX127X_PARAM_PASELECT` define instead of `sx127x_params[0].paselect`? That would be cleaner.

Reply:
I get bored because of compiler's complaining so I ended up doing this... I'll fix.